draft-packages/form/KaizenDraft/Form/RadioGroup/RadioGroup.spec.tsx

```diff
@@ -66,21 +14,13 @@ describe("<RadioGroup /> ", () => {
     })
   })
   describe("accessibilty", () => {
-    it("should markup aria label when supplied labelId", () => {
-      const labelId = "labelId"
-      const { getByText, getByRole } = render(
-        <RadioGroup
-          labelText="Label title"
-          labelId={labelId}
-          automationId="test"
-        />
-      )
-      expect(getByRole("radiogroup").getAttribute("aria-labelledby")).toEqual(
-        labelId
-      )
-      expect(
-        getByText("Label title").parentElement?.getAttribute("id")
-      ).toEqual(labelId)
+    it("should have an accessible name when not provided a labelId", () => {
```

Comment:
Two minor things here:
1. I've been removing all `shoulds` from tests. We are very inconsistent with it, and when I learned testing I was taught not to use "shoulds" because "you assert that the code does do a thing, not that is should be doing it".
2. The way you're using `expect` without a matcher will work - even without the `expect` it will work, because if it can't find the element `getBy...` will throw and hence the test will fail. But for the sake of readability and clarity I'd use the `.tobeInTheDocument()` matcher here.